server/dash/dashboards/nc_summary_comparison.py

```diff
@@ -80,32 +78,29 @@ def generate_summary_dropdowns():
     Return:
         Dash html div element containing nc and request type dropdowns.
     """
-    dropdowns = html.Div(children=[
-        html.Div(dcc.Dropdown(sorted([n for n in set(api_data_df["councilName"])]),
-                " ", id="nc_dropdown",
+    return html.Div(children=[
+        html.Div(dcc.Dropdown(sorted(list(set(api_data_df["councilName"]))),
+                placeholder=" ", id="nc_dropdown",
```

Comment:
sorry I just used `placeholder` as an example but I didn't make it clear. Is this supposed to be `value`? https://dash.plotly.com/dash-core-components/dropdown

Reply:
Thanks for the catch - resolved.